app/configuration.py

```diff
@@ -0,0 +1,39 @@
+from pydantic import BaseSettings, root_validator
+
+
+class DD20Settings(BaseSettings):
+    """BaseSetting superclass for managing configuration"""
+
+    debug: bool = False
+    use_mock_data: bool = False
+    mock_dd20_filepath: str = "tests/valid-testdata/DD20.xlsm"
+    mock_dd20_mapping_filepath: str = "tests/valid-testdata/Limits_other.xlsx"
+    mock_mrid_mapping_filepath: str = "tests/valid-testdata/seg_liine_mrid_PROD.csv"
+    dd20_filepath: str = "/input/DD20.XLSM"
+    dd20_mapping_filepath: str = "/input/Limits_other.xlsx"
+    mrid_mapping_filepath: str = "/input/seg_line_mrid_PROD.csv"
+    station_data_valid_hash: str
+    line_data_valid_hash: str
+    api_port: int = 5000
+    api_dbname: str = "CONDUCTOR_DATA"
+    api_refresh_rate: float = 60
+
+    @root_validator(pre=False)
+    def assign_mock_data(cls, values):
+
+        assert "use_mock_data" in values, "use_mock_data is invalid or missing"
+
+        use_mock_data = bool(values["use_mock_data"])
+        if use_mock_data:
+            values["dd20_filepath"] = values["mock_dd20_filepath"]
+            values["dd20_mapping_filepath"] = values["mock_dd20_mapping_filepath"]
+            values["mrid_mapping_filepath"] = values["mock_mrid_mapping_filepath"]
+
+        return values
+
+    # Configure how envionment .env file should be parsed
+    class Config:
+        env_file = "./app/.env"
```

Comment:
```suggestion
        env_file = "app/.env"
```
"./" just references the current folder so should not be necessary.
I will say though - i am uncertain how pydantic reads the path, and if it does so like standard python (i guess it does) it will start in cwd, which could make the relative path you have provided non-functional. So perhaps:
```suggestion
        import os # Move me to the top!
        env_file = os.path.join(os.path.split(os.path.abspath(__file__))[0], ".env")
```